raiden/network/rpc/client.py

```diff
@@ -1039,8 +1039,10 @@ def __init__(
         version = web3.clientVersion
         supported, eth_node, _ = is_supported_client(version)
 
-        if not supported or eth_node is None:
-            raise EthNodeInterfaceError(f"Unsupported Ethereum client {version}")
+        if eth_node is None or supported == VersionSupport.UNSUPPORTED:
+            raise EthNodeInterfaceError(f'Unsupported Ethereum client "{version}"')
+        if supported == VersionSupport.WARN:
+            log.warn(f'Unsupported Ethereum client version "{version}"')
```

Comment:
It might make sense to be a bit more specific here (e.g. unsupported higher version but we'll continue anyway). 
And maybe output the expected and found version.